src/main/java/com/bunq/tinker/libs/BunqLib.java

```diff
@@ -299,4 +315,27 @@ private SandboxUser generateNewSandboxUser() {
       throw new BunqException(e.getMessage());
     }
   }
+
+  private void requestSpendingMoneyIfNeeded() {
+    if (shouldRequestSpendingMoney()) {
+      RequestInquiry.create(
+        new Amount(REQUEST_SPENDING_MONEY_AMOUNT, CURRENCY_EUR),
+        new Pointer(POINTER_TYPE_EMAIL, REQUEST_SPENDING_MONEY_RECIPIENT),
+        REQUEST_SPENDING_MONEY_DESCRIPTION,
+        false
+      );
+
+      try {
+        Thread.sleep(REQUEST_SPENDING_MONEY_WAIT_TIME_MILLISECONDS);
+      } catch (InterruptedException exception) {
+        // We don't care about being interrupted.
```

Comment:
This is risky and makes debug hard. So just print the error message instead of completely ignoring it :D